project/ws/app/src/lib/routes/events/routes/list/list-event.component.ts

```diff
@@ -221,7 +224,7 @@ export class ListEventComponent implements OnInit, AfterViewInit, OnDestroy {
         const formatedDate = new Date(year, month, date, hours, minutes, seconds, 0)
         // let format = 'YYYY-MM-DD hh:mm a'
         // if (!timeAllow) {
-        const format = 'YYYY-MM-DD'
+        const format = 'MMM DD, yyyy'
```

Comment:
as per discussion date format should be dd-mm-yyyy 
can you please check and confirm on same